LabTools/CoreWidgets/RemoteEnableWidget.py

```diff
@@ -0,0 +1,386 @@
+"""
+Created Aug 13th, 2019
+
+@author: zackorenberg
+
+This widget allows the user to set a default Remote Enable State, as well as assert specific modes to specific devices in real time
```

Comment:
Maybe here you can describe a bit more in details and/or provide sources/manuals for interested readers/future users :)

Reply:
I have updated it in the next commit. I could not find the website that precisely stated each level, but I copy-pasted it verbatim into the widget. (It took me a few hours to find and understand what exactly occurs for each level, piecing together information found in NI's limited documentation for their control_ren(mode) method, forum questions, and random user manuals for other 488.2 devices). Let me know if I should add anything else.

Reply:
> It took me a few hours to find and understand what exactly occurs for each level

Finding information and crunching it out in a more understandable format is very useful and can be very time consuming, thank you for taking this time :)!

Reply:
> I have updated it in the next commit. 

Did you push the commit? (FYI, if you copy the commit number in any github thread, it will automatically create a link to the commit ^^)